### DISCUSSION TITLE
The team using the 'pro' plan is not linked to their GitHub account, so the latest commits are not being reflected

### DISCUSSION THREAD

**Discussion by moojecho** [vercel/community#6439] [Help]
### Summary

As the title says, a team using the 'pro' plan is not linked to GitHub, and the commits are not being reflected. I am the owner of this team and I am using the hobby plan, and my account is properly connected to GitHub. Please let me know how to resolve this issue.

![image](https://github.com/vercel/community/assets/107379879/6590dd39-a27f-4bc0-bec1-61f2ccc330c6)

The situation is such that access is not available as shown in the picture.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/107379879/2449ade7-5de6-409b-9864-b9d318389b81)

**Comment by amyegan** [comment]
Hey @moojecho. This looks like a permission issue between your Vercel account and GitHub repo. Here are a few things you can try:

- Make sure your Vercel account email and [git commit email](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) match
- Check the app permissions for Vercel at https://github.com/settings/installations, and make sure Vercel has repository access
- Disconnect and reconnect your GitHub account from your Vercel dashboard, which will force the connection to reset

**Comment by moojecho** [comment]
The mentioned aspects are already properly applied. Even the part where I mentioned 'If you disconnect your GitHub account from the Vercel dashboard and reconnect it, the connection will be forcibly reset' has already been tried. As mentioned above, the account itself is properly connected to GitHub. However, the connection with GitHub is not working properly only for the team using the pro plan.


![image](https://github.com/vercel/community/assets/107379879/99820784-82f1-4e9e-a31f-92e3871c7e48)
![image](https://github.com/vercel/community/assets/107379879/ba3021f6-99a4-4ce5-9f35-7cf40d7cf5e3)

![image](https://github.com/vercel/community/assets/107379879/7e7fae88-3679-44c1-b553-25ff30ab7a33)
